src/input/datetime.rs

```diff
@@ -447,7 +449,7 @@ pub fn float_as_duration<'a>(input: &'a impl Input<'a>, total_seconds: f64) -> V
 #[pyclass(module = "pydantic_core._pydantic_core", extends = PyTzInfo)]
 #[derive(Clone)]
 #[cfg_attr(debug_assertions, derive(Debug))]
-struct TzInfo {
+pub(crate) struct TzInfo {
```

Comment:
```suggestion
pub struct TzInfo {
```
If we're exporting it, may as well make it rust public too to match other things.